### ISSUE TITLE
[Global] Decorative SVG missing attributes

### ISSUE THREAD

**Issue by svinkle** [CovidShield/portal#10]
## Issue summary

Each of the decorative SVG elements are missing attributes to help convey/remove their semantic meaning for AT.

- IE/Edge has a bug where SVG as a child element of focusable elements create an extra tab stop.
- Some assistive technology can still access SVG elements via internal navigation methods creating an invisible tab-stop.

<img width="471" alt="Screen Shot 2020-05-18 at 11 04 37 AM" src="https://user-images.githubusercontent.com/1392632/82245117-5ef71900-9910-11ea-8105-13fda3c7a0f1.png">
<img width="217" alt="Screen Shot 2020-05-18 at 2 03 45 PM" src="https://user-images.githubusercontent.com/1392632/82245198-80f09b80-9910-11ea-825f-6068ee7ad7e6.png">

#### HTML

```html
<svg class="icon"><use xlink:href="#question-mark"></use></svg>

<!-- … -->

<svg class="popover__icon"><use xlink:href="#users"></use></svg>

<!-- … -->
```

## Steps to reproduce

1. Go to https://portal-rails.myshopify.io/ with with any supported browser
2. Login with `admin`/`password`
3. Use the web inspector to review the SVG markup

### Expected

- Screen readers to bypass decorative SVG, AT navigation will commence as expected.

### Actual

- Sometimes AT may announce the SVG, navigation requires extra effort.

## Recommendation

1. Add `focusable="false"` to SVG elements embedded within focusable elements to avoid the Edge bug.
2. Add `aria-hidden="true"` and for any lone/solo SVG icons in order to remove the SVG semantics.

#### HTML

```html
<svg class="icon" focusable="false" aria-hidden="true">
  <use xlink:href="#question-mark"></use>
</svg>

<!-- … -->

<svg class="popover__icon" focusable="false" aria-hidden="true">
  <use xlink:href="#users"></use>
</svg>

<!-- … -->
```

## Specifications

- **Component:** Global
- **WCAG Principle:** Operable
- **WCAG SC:** 2.1.1 Keyboard
- **Severity:** Low
- **Effort:** Low
